### PR TITLE
Add docs for Z-Wave JS integration

### PR DESCRIPTION
**Pull request by cgarwood**
## Proposed change

Add docs for the Z-Wave JS integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/45020
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/2151
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by MartinHjelmare — DISMISSED** [review]

File: source/_integrations/zwave_js.markdown [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+## Requirements
+
+### Core installation
+
+- The [zwave-js-server](https://github.com/zwave-js/zwave-js-server) installed and running in your network.
+
+### Hardware requirements
+
+- Supported Z-Wave dongle. Most dongles compatible with OpenZWave will work with Z-Wave JS. See this [list](/docs/z-wave/controllers/#supported-z-wave-usb-sticks--hardware-modules) of controllers. The Z-Wave controller dongle should be connected to the same host as where the the Z-Wave JS server is running.
```
Comment (balloob) [inline source/_integrations/zwave_js.markdown]:
```suggestion
- [Supported Z-Wave dongle](/docs/z-wave/controllers/#supported-z-wave-usb-sticks--hardware-modules). The Z-Wave controller dongle should be connected to the same host as where the the Z-Wave JS server is running.
```

File: source/_integrations/zwave_js.markdown [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+Click on the `+` sign to add an integration and click on **Z-Wave JS**.
+After completing the configuration flow, the Z-Wave JS integration will be
+available.
+
+## Current Limitations
+
+As this integration is still in the early stages there are some important limitations to be aware of.
+
+- Platform support is currently limited to basic `sensor` and `light` entities.
+- Setting a network key is not currently supported by [zwave-js-server](https://github.com/zwave-js/zwave-js-server).
```
Comment (balloob) [inline source/_integrations/zwave_js.markdown]: This is supported. You can pass a path to a configuration file. https://github.com/zwave-js/zwave-js-server#start-server

File: source/_integrations/zwave_js.markdown [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  - Sensor
+ha_release: '2021.2'
+ha_iot_class: Local Push
+ha_config_flow: true
+ha_codeowners:
+  - '@home-assistant/z-wave'
+ha_domain: zwave_js
+---
+
+This integration allows you to utilize Z-Wave JS to control a Z-Wave network.
```
Comment (balloob) [inline source/_integrations/zwave_js.markdown]:
```suggestion
This integration allows you to control a Z-Wave network via Z-Wave JS.
```

File: source/_integrations/zwave_js.markdown [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+## Requirements
+
+### Core installation
+
+- The [zwave-js-server](https://github.com/zwave-js/zwave-js-server) installed and running in your network.
+
+### Hardware requirements
+
+- [Supported Z-Wave dongle](/docs/z-wave/controllers/#supported-z-wave-usb-sticks--hardware-modules). The Z-Wave controller dongle should be connected to the same host as where the the Z-Wave JS server is running.
```
Comment (frenck) [inline source/_integrations/zwave_js.markdown]:
```suggestion
- [Supported Z-Wave dongle](/docs/z-wave/controllers/#supported-z-wave-usb-sticks--hardware-modules). The Z-Wave controller dongle should be connected to the same host as where the Z-Wave JS server is running.
```

**Review by balloob — APPROVED** [review]

**Pull request merged**